src/views/home/Home.vue

```diff
@@ -28,50 +28,79 @@
 </template>
 
 <script lang="ts">
-import Vue from 'vue';
+// import Vue from 'vue';
+import { Component, Prop, Emit, Vue, Watch } from 'vue-property-decorator';
 import Waiting from '@/components/misc/Waiting.vue';
 import { EditionInfo } from '@/models/edition';
 import PersonalEditions from './components/PersonalEditions.vue';
 import PublicEditions from './components/PublicEditions.vue';
 
-export default Vue.extend({
-    name: 'home',
-    components: {
+
+@Component({
+  name: 'home',
+  components: {
         Waiting,
         PersonalEditions,
-        PublicEditions,
-    },
-    data() {
-        return {
-            filter: '',
-            personalEditionsCount: 0,
-            publicEditionsCount: 0,
-        };
-    },
-
-    created() {
+        PublicEditions
+  }
+})
+
+export default class Home extends Vue {
+
+    // component data
+    // =====================
+
+    private filter: string = '';
+    private personalEditionsCount: number = 0 ;
+    private publicEditionsCount: number = 0 ;
+
+    // hooks as constructor
+    // ========================
+    protected created() {
         this.$state.prepare.allEditions();
```

Comment:
Moving this from mounted to created did not fix the actual problem, it just made it less likely to happen. We have a race condition - if prepare.allEditions finishes before the component is rendered, all is well. If it does not, we have errors. So I suggest we solve this properly - add a flag that says the editions have been prepared. In mounted, or created, set the flag to false, await the prepare function and then set it to true. Use it as a conditional in the html, so you don't render anything based on the editions until they're ready